tests/foreman/cli/test_errata.py

```diff
@@ -1091,22 +1092,28 @@ def test_positive_list_filter_by_product_id_and_org_label(self):
             set([])
         )
 
-    @stubbed()
-    def test_positive_list_filter_by_product_name(self):
```

Comment:
@oshtaier, please look at http://projects.theforeman.org/projects/katello/repository/hammer-cli-katello/revisions/31d103f9d672907859bee8d3c09857578ab2365c/entry/lib/hammer_cli_katello/erratum.rb
to see how this bug has been resolved